File: work-spring/day0903_jwt/src/main/java/com/grepp/jwt/model/service/UserService.java
package com.grepp.jwt.model.service;

import com.grepp.jwt.model.entity.UserEntity;
import com.grepp.jwt.model.repository.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class UserService {
    @Autowired
    private UserRepository repository;

    public UserEntity join(UserEntity entity){
        return repository.save(entity);
    }

    public UserEntity getByCredentials(String username, String password){
        return repository.findByUsernameAndPassword(username,password);
    }
}

File: work-kotlin/day1021_todo/src/main/kotlin/com/grepp/todo/model/service/UserService.kt
package com.grepp.todo.model.service

import com.grepp.todo.model.entity.UserEntity
import com.grepp.todo.model.repository.UserRepository
import org.springframework.stereotype.Service

@Service
class UserService(val repo:UserRepository) {
    // 회원가입
    fun join(entity:UserEntity) = repo.save(entity)
    // 로그인
    fun login(username:String, password:String) = repo.findByUsernameAndPassword(username, password)
}